util/unifyfs/src/unifyfs-rm.c

```diff
@@ -483,58 +485,120 @@ static int slurm_read_resource(unifyfs_resource_t* resource)
 static size_t construct_server_argv(unifyfs_args_t* args,
                                     char** server_argv)
 {
-    size_t argc;
+    size_t argc = 0;
     char number[16];
 
-    if (server_argv != NULL) {
-        if (args->server_path != NULL) {
-            server_argv[0] = strdup(args->server_path);
-        } else {
-            server_argv[0] = strdup(BINDIR "/unifyfsd");
-        }
-    }
-    argc = 1;
-
-    if (args->debug) {
+    switch (args->arg_type) {
+    case UNIFYFS_CONSTRUCT_ARGS_UNIFYFSD:
         if (server_argv != NULL) {
-            server_argv[argc] = strdup("-v");
-            snprintf(number, sizeof(number), "%d", args->debug);
-            server_argv[argc + 1] = strdup(number);
+            if (args->server_path != NULL) {
+                server_argv[0] = strdup(args->server_path);
+            } else {
+                server_argv[0] = strdup(BINDIR "/unifyfsd");
```

Comment:
Should we be hard coding the path to the `unifyfsd` here?

Reply:
No idea.  I didn't do that.  It was that way.  Mike talked about this on the call on Monday, that if the unifyfsd isn't specified by flag, there's a default location for it.

Reply:
Yes, the BINDIR path to the server is only used as a default. If the user gives a path on the command-line, that's the 'if' case.

Reply:
What if the user has `unifyfsd` in their `$PATH`?

Reply:
Not sure this is related as I'm still lacking some experience here, but is what @tonyhutter is saying similar to what Spack does? Doing the commands:
```
spack install unifyfs
spack load unifyfs
```
adds the appropriate directories to `PATH`, `MANPATH`, `CPATH`, and LD_LIBRARY_PATH`.

As a user, after doing `spack load unifyfs`, the commands `unifyfs` and `unifyfsd` are recognized (i.e., can simply type `unifyfs start -h` or `unifyfsd -h`).